modules/platform/nuxeo-platform-task/nuxeo-platform-task-api/src/main/java/org/nuxeo/ecm/platform/task/TaskService.java

```diff
@@ -254,14 +255,31 @@ String endTask(CoreSession coreSession, NuxeoPrincipal principal, Task task, Str
      *
      * @since 5.7.3
      */
-    void reassignTask(CoreSession session, String taskId, List<String> actors, String comment);
+    default void reassignTask(CoreSession session, String taskId, List<String> actors, String comment) {
+        reassignTask(session,taskId,actors,comment,new HashMap<>());
+    }
+
+    /**
+     * @since 11.5
+     */
+    void reassignTask(CoreSession session, String taskId, List<String> actors, String comment,
```

Comment:
Could you add a `@see #reassignTask(...)` with the right parameters referencing the other method? Your IDE should help to complete them.